task/04-arrays-tasks.js

```diff
@@ -70,11 +77,17 @@ function doubleArray(arr) {
  *    [] => [] 
  */
 function getArrayOfPositives(arr) {
-   throw new Error('Not implemented');
+   let positivesArr = arr.reduce( (newArr,number) => { 
+      if (number > 0) {
+         newArr.push(number);
+         }
+         return newArr;
+   }, []);
+   return positivesArr;
```

Comment:
Please, use single filter method  to solve this task

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter